### ISSUE TITLE
yargs.showHelp() is not working

### ISSUE THREAD

**Issue by paulintrognon** [bcoe/yargs#143]
Hello,

It appears that the `.showHelp` is not working for me... If do exactly as told in the doc:

```
var yargs = require("yargs")
       .usage("$0 -operand1 number -operand2 number -operation [add|subtract]");
yargs.showHelp();
```

I get:

```
TypeError: Cannot read property 'newAliases' of undefined
    at Object.self.help (/home/paulin/projet/node_modules/yargs/lib/usage.js:137:45)
    at Object.self.showHelp (/home/paulin/projet/node_modules/yargs/lib/usage.js:237:29)
    at Object.Argv.self.showHelp (/home/paulin/projet/node_modules/yargs/index.js:304:15)
    at Object.<anonymous> (/home/paulin/projet/test.js:3:7)
```

**Comment by paulintrognon** [comment]
OK, I ran some tests and realized I had to "call" .argv in order to make it work, like so:

```
var yargs = require("yargs")
       .usage("$0 -operand1 number -operand2 number -operation [add|subtract]");
yargs.argv;
yargs.showHelp();
```

The problem then occur if you call `.showHelp();` *before* `.argv`. I imagine this is because things are settled only once you've called `.argv`?

Can the documentation explain that or can you make the example work? Thank you =)

**Comment by bcoe** [comment]
@paulintrognon I'd like to fix this, as with `.help()` the solution will be calling:

```        if (!self.parsed) parseArgs(processArgs); // run parser, if it has not already been executed.```

When `showHelp()` is executed, I'll patch this ASAP.

**Comment by bcoe** [comment]
@paulintrognon this bug should now be fixed in `yargs@3.7.1`, please let me know if you continue seeing problems.
